.github/workflows/codeql-analysis.yml

```diff
@@ -6,7 +6,7 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ "main", experimental ]
+    branches: [ "main", experimental, "feature*" ]
```

Comment:
I'm curious if the `experimental` here is automatic wildcard or not (I'm assuming the intention was to apply to `experimental-huggingface` and `experimental-typescript` and others? Would be nice to be conistent.

Reply:
Let's evaluate changes to 'experimental' after this change comes in.